docs/source/60_building_apps.md

```diff
@@ -256,6 +256,58 @@ $ spark-submit \
 A complete example of this pattern can be found in the
 [heuermh/adam-examples](https://github.com/heuermh/adam-examples) repository.
 
+### Using ADAM’s RegionJoin API
+Another useful API implemented in ADAM is the RegionJoin API, which joins two
+genomic datasets that contain overlapping regions. This primitive is useful for
+a number of applications including variant calling (identifying all of the reads 
+that overlap a candidate variant), coverage analysis (determining the coverage 
+depth for each region in a reference), and INDEL realignment (identify INDELs 
+aligned against a reference).
+
+There are two overlap join implementations available in ADAM: 
+BroadcastRegionJoin and ShuffleRegionJoin. The result of a ShuffleRegionJoin 
+is identical to the BroadcastRegionJoin, however they serve different 
```

Comment:
I would add that the shuffle region join supports a few more operations (e.g., full outer join).